### ISSUE TITLE
Private call invalid timestamp

### ISSUE THREAD

**Issue by Serpens66** [sammchardy/python-binance#176]
Hi :)

How to create an order with your script?
I did the following in python 3.4:

b = Client(key,secret)
a = b.create_test_order(symbol="LTCUSDT",
            side="SELL",
            type="LIMIT",
            timeInForce="GTC",
            quantity= 0.2,
            price= "200")
print(a)

But it always returns the 400 response and I don't know why =/

**Comment by sammchardy** [comment]
Hi @Serpens66 what is the error message that comes back?

As a test order that works perfectly fine for me.

**Comment by Serpens66** [comment]
I disabled the exception handling of your script, that's why I only saw response 400, but not the message from API. I enabled it again and this API message:

{"code":-1021,"msg":"Timestamp for this request was 1000ms ahead of the server's time."}

This happens for all private calls, guess there is no solution except to adjust my system time?! Or maybe reduce the timestamp by few seconds...

**Comment by ennethouse-darhsic** [comment]
Hi @Serpens66, I am getting the same error. Where did you add the "-10000"? I see client.py line 155. Anywhere else?

@sammchardy - Mine works perfectly fine in Windows 10, but am getting the error for all private calls in Linux.

**Comment by Serpens66** [comment]
@ennethouse-darhsic As I wrote above, I made the calculation:
get_server_time() - time.time() 
and notced that my system time (time.time) is 9.5 seconds ahead of the time of binance.
Therefore I substracted 10 seconds (10000) from the timestamp to see if this helps and yes it does.

The permanent and overall solution is written in my last sentence in previous post. Automate the calculation of the time difference and substrct it when getting the timestamp.
